include/cudaq/Optimizer/CodeGen/Passes.h

```diff
@@ -31,15 +31,16 @@ void registerConvertToQIRPass();
 /// @param pm Pass Manager to add QIR passes to
 /// @param convertTo Expected to be "qir-base" or "qir-adaptive" (comes from the
 /// cudaq-translate command line `--convert-to` parameter)
-void addQIRProfilePipeline(mlir::OpPassManager &pm, llvm::StringRef convertTo);
+void addQIRProfilePipeline(mlir::OpPassManager &pm,
+                           const std::string &convertTo);
```

Comment:
Were you seeing the problem with StringRef in the regression tests or was it some other testing?

Reply:
> Were you seeing the problem with StringRef in the regression tests or was it some other testing?

It was in the regression tests. [Here](https://github.com/NVIDIA/cuda-quantum/actions/runs/6503176516/job/17663385026) is the run. It was only on arm64/gcc12. All other platforms worked without this change.